src/main/User/UserController.java

```diff
@@ -271,4 +279,35 @@ public static JSONObject mergeJSON(
     }
     return merged;
   }
+
+  public InputStream getUserPfp(String username) {
```

Comment:
can we move this into a service and not let this sit in the usercontroller? it just makes it hard to test later on

Reply:
it was never used ooops.  I already moved to a service. I just deleted it.